src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -23,7 +23,7 @@ public class ScriptProjectSystem : IProjectSystem
     {
         // aligned with CSI.exe
         // https://github.com/dotnet/roslyn/blob/version-2.0.0-rc3/src/Interactive/csi/csi.rsp
-        internal static readonly IEnumerable<string> DefaultNamespaces = new[]
+        private static readonly IEnumerable<string> DefaultNamespaces = new[]
```

Comment:
Note that the build failed because you made this private. It's accessed by ScriptContextModel.cs [here](https://github.com/OmniSharp/omnisharp-roslyn/blob/dev/src/OmniSharp.Script/ScriptContextModel.cs#L13).